aospy/data_loader.py

```diff
@@ -91,7 +92,11 @@ def set_grid_attrs_as_coords(ds):
     Dataset
         Dataset with grid attributes set as coordinates
     """
-    int_names = GRID_ATTRS.keys()
+    if not set_time_vars:
```

Comment:
Slight preference to use the affirmative, i.e. `if set_time_vars:` as the main `if` statement, with the `else` corresponding to `not set_time_Vars`